### PR TITLE
Events reinvented

### PR DESCRIPTION
**Pull request by bjanssen-pon**

### CODE REVIEW

File: chapters/events.adoc [hunk 1]
```diff
@@ -166,152 +166,73 @@ EventType:
...
-        - external-public
+        1.0
+        1.1
+        2.0
+    type: 
       description: |
-        Intended target audience of the event type, analogue to audience definition for REST APIs 
-        in rule #219 https://github.com/PonDigitalSolutions/restful-api-guidelines
-    owning_application:
+        timestamp
```
Comment (apiest-pon) [inline chapters/events.adoc:186]: ? prob. event?

File: chapters/events.adoc [hunk 2]
```diff
@@ -166,152 +166,73 @@ EventType:
...
+      example: |
+        com.github.pull.create
+        exactonline.order.update
+        com.swapfiets.order.create
+        com.gazelle.delivery.update
+    source:
       description: |
-        Name of the application (eg, as would be used in infrastructure
-        application or service registry) owning this `EventType`.
+        timestamp
```
Comment (apiest-pon) [inline chapters/events.adoc:195]: Category?

File: chapters/events.adoc [hunk 3]
```diff
@@ -166,152 +166,73 @@ EventType:
...
-          Indicates a single ordering field. It is a simple path (dot separated) to
-          the JSON leaf element of the whole event object, including the contained metadata and data (in
-          case of a data change event) objects. It must point to a field of type
-          string or number/integer (as for those the ordering is obvious), and must be
-          present in the schema.
-        example: "data.order_change_counter"
-    ordering_instance_ids:
-      type: array
+        OPTIONAL. This is the Timestamp of the created context. Can be used to compare events which have the same subject involved.
+      type: string
```
Comment (apiest-pon) [inline chapters/events.adoc:220]: add format: date-time
https://swagger.io/docs/specification/data-models/data-types/#string

**Review by apiest-pon — COMMENTED** [review]
perhaps delete the .DS_Store file?

create a hyperlink or mention Zalando when Nakadi is mentioned

File: chapters/events.adoc (hunk 2 above)
Comment (bjanssen-pon) [reply chapters/events.adoc:195]: Moest inderdaad de description nog goed vullen

File: chapters/events.adoc (hunk 1 above)
Comment (bjanssen-pon) [reply chapters/events.adoc:186]: Moest inderdaad de description nog goed vullen

File: chapters/events.adoc (hunk 3 above)
Comment (bjanssen-pon) [reply chapters/events.adoc:220]: In the event its a string typed object, but it represents a datetime object.

Comment (apiest-pon) [reply chapters/events.adoc:220]: yes a date-time string, so add format: date-time

**Review by apiest-pon — APPROVED** [review]

**Review by w3nl — APPROVED** [review]

**Pull request merged**